test/unit/test_tag_gmsh.py

```diff
@@ -0,0 +1,325 @@
+import numpy as np
```

Comment:
Please add short documentation of what the module is testing. This we should really for all tests.